extensions/mssql/src/utils.ts

```diff
@@ -132,6 +133,17 @@ export async function getParallelMessageProcessingConfig(): Promise<boolean> {
 	return (azdata.env.quality === azdata.env.AppQuality.dev && setting.globalValue === undefined && setting.workspaceValue === undefined) ? true : config[parallelMessageProcessingConfig];
 }
 
+export async function getEnableSqlAuthenticationProviderConfig(): Promise<boolean> {
+	const config = getConfiguration();
+	if (!config) {
+		return false;
+	}
+	const setting = config.inspect(enableSqlAuthenticationProviderConfig);
+	// For dev environment, we want to enable the feature by default unless it is set explicitely.
+	// Note: the quality property is not set for dev environment, we can use this to determine whether it is dev environment.
```

Comment:
This shouldn't be true anymore - I had it default to dev so there should always be a value for this. 

https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/api/common/sqlExtHost.api.impl.ts#L623

Looks like I just forgot to update the comment above. Could you verify this is true and then remove these comments if so?